README.md

```diff
@@ -86,6 +86,14 @@ pcap = { version = "1", features = ["capture-stream"] }
 
 **This feature is supported only on Linux and Mac OS X. It will not work and is not supported on Windows.**
 
+## Unstable Features
+
+Use at your own risk, we do not considered this our public API yet.
+
+### `lending-iter`
+
+This use `gat-std` crate to have a common lending iterator, this crate is not a lot used so this may change later. See `lendingiterprint` example.
```

Comment:
No need for so much detail. For consistency with the description of `capture-stream`, please change the description to something like:

Use the `lending-iter` feature to enable the lending packet iterator. See the `lendingiterprint` example.